src/Chapter2.hs

```diff
@@ -740,7 +752,8 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: Eq a => a -> [[a]] -> [[a]]
+contains x xss = filter (\xs -> elem x xs) xss
```

Comment:
Now, after you've mastered eta-reduction, you can apply such a technique to other functions as well  🙂 
But your implementation is already great 💯